lib/cosmos/tools/tlm_viewer/screen.rb

```diff
@@ -221,6 +221,10 @@ def initialize(full_name, filename, notify_on_close = nil, mode = :REALTIME, x_p
       # regular TlmViewer application
       @single_screen = single_screen
 
+      # Read the application wide stylesheet if it exists
+      app_style = File.join(Cosmos::USERPATH, 'config', 'tools', 'application.css')
```

Comment:
After starting to write a bunch of stylesheets I quickly realized that having an application wide style would be really nice.